helpers/executeCommand.js

```diff
@@ -4,26 +4,27 @@ const shelljs = require('shelljs');
  * A helper function that executes a single shell command or JavaScript function.
  * Supports asynchronous JS functions, and executes shell commands in a child process
  * so that the event loop is not blocked while executing the command.
- * @param  {string | function(): (number | Promise<number>)} command
- * @return Exit code for the executed command, a non-zero value indicates failure
+ * @param  {string | function(): (void | Promise<void>)} command
+ * @return {Promise<void>} A promise that resolves if the command was succesfull, otherwise throws an `Error`. If the command is a shell command, the error message will be stderr of the command.
```

Comment:
Would be good if we breakup long comment lines into multiple lines and lint against that.

Reply:
I thought Prettier would take care of it, but it looks like it does not touch comments.